packages/apps/docs/src/docs/build/authentication/create-transactions.md

```diff
@@ -134,68 +117,69 @@ a Kadena SpireKey wallet might look like for the `n_eef68e581f767dd66c4d4c39ed92
+1. Stringify the unsigned transaction (tx) you've prepared:
+   
+   ```typesecript
+   const encodedTx = btoa(JSON.stringify(tx));
+   ```
+   
+2. Use the `encodeURIComponent` function to encode the return URL:
 
-The following is an example of how you would construct the route:
+   ```typescript
+   const encodedReturnUrl = encodeURIComponent(RETURN_URL);
+   ```
 
-```ts
-// tx is the unsigned transaction you constructed
-const encodedTx = btoa(JSON.stringify(tx));
+1. Construct the request URL using the `encodedTx` and `encodedReturnUrl` to send the transaction to Kadena SpireKey:
+   
+   ```typescript
+   const sendTransactionUrl = `https://spirekey.kadena.io/sign#transaction=${encodedTx}&returnUrl=${encodedReturnUrl}`;
+   ```
 
-// Use`encodeURIComponent` so that the return url is still readable
-const encodedReturnUrl = encodeURIComponent(RETURN_URL);
+Because transaction strings can be longer than what is accepted in `searchParameters`, you should send transactions to the `/sign` endpoint using the anchor hashtag (`#`) instead of the `searchParameters` question mark (`?`). 
 
-// The url you need to navigate to sign and return the transaction
-const sendTransactionUrl = `https://spirekey.kadena.io/sign?transaction=${encodedTx}&returnUrl=${encodedReturnUrl}`;
-```
+For example, a signature request might look similar to this:
+
+https://spirekey.kadena.io/sign#transaction=encodedTx&returnUrl=www.mydapp.com
+
+The following table describes the parameters that you can include in your signing request:
 
-After you construct the route to the wallet with the required parameters, you
-can navigate to the wallet to handle the signing.
+| Parameter | Type | Required | Description |
+| --------- | ---- | -------- | ----------- |
+| `transaction` | string  | Required | A base64 encoded string of the unsigned transaction. |
+| `returnUrl` | string  | Required | The url, encoded as a uriComponent, that the Kadena SpireKey should redirect users to after they have signed the transaction. |
+| `translations` | string  | Optional | Custom descriptions that explain what capabilities or operations that the user is signing for. |
+| `optimistic` | boolean | Optional | Allows applications to continue transaction workflows without having to wait for the transaction to be confirmed on the blockchain. When this parameter is included, `pendingTxIds` are returned so that the application can keep track of the status of the submitted transactions and update the UI accordingly. For more information about the optimistic transaction flow, see [Optimistic account onboarding](/build/authentication/integrate#optimistic-account-onboardingh-416162207). |
+
+After you construct the request with the required parameters, you can connect to Kadena SpireKey to enable the user to sign the transaction.
```

Comment:
"connect" could be confusing here, because it also part of the SpireKey API (connecting an account). Maybe "redirect".